components/beebole_app/beebole_app.app.mjs

```diff
@@ -1,11 +1,60 @@
+import { axios } from "@pipedream/platform";
+
 export default {
   type: "app",
   app: "beebole_app",
-  propDefinitions: {},
+  propDefinitions: {
+    companyId: {
+      type: "string",
+      label: "Company ID",
+      description: "The ID of the company",
+      async options() {
+        const response = await this.manageCompanies({
+          data: {
+            service: "company.list",
+          },
+        });
+        const companiesIDs = response.companies;
+        return companiesIDs.map(({
+          id, name,
+        }) => ({
+          value: id,
+          label: name,
+        }));
+      },
+    },
+    companyName: {
+      type: "string",
+      label: "Company Name",
+      description: "The name of the company",
+    },
+  },
   methods: {
-    // this.$auth contains connected account data
-    authKeys() {
-      console.log(Object.keys(this.$auth));
+    _baseUrl() {
+      return "https://beebole-apps.com/api/v2";
+    },
+    async _makeRequest(opts = {}) {
+      const {
+        $ = this,
+        data,
```

Comment:
I think `data` does not need to be destructured here and can be left implicit within `otherOpts`